tests/testthat/test-pivot-long.R

```diff
@@ -267,19 +267,19 @@ test_that("no names doesn't generate names (#1120)", {
 test_that("multiple names requires names_sep/names_pattern", {
   df <- tibble(x_y = 1)
 
-  expect_snapshot({
-    (expect_error(build_longer_spec(df, x_y, names_to = c("a", "b"))))
+  expect_snapshot(error = TRUE, {
+    build_longer_spec(df, x_y, names_to = c("a", "b"))
```

Comment:
I tend to use 1 `expect_snapshot()` per expression that errors. Otherwise you don't know if both error, you just know that at least 1 errored

The previous `expect_error()` wrappers captured and then reemitted the individual errors, so previously it ensured that each expression did indeed error

Reply:
Thanks for the tip! Done. I ended up doing this across the board, even for snapshots I disn't touch before.